roles/moodle/templates/moodle-nginx.conf.j2

```diff
@@ -1,13 +1,32 @@
-location /moodle {
-    proxy_set_header X-Real-IP  $remote_addr;
-    proxy_set_header X-Forwarded-For $remote_addr;
-    proxy_set_header Host $host;
-    proxy_pass http://127.0.0.1:{{ apache_port }};
+# 2021-07-02: Let's monitor & learn from these 2 pages year-by-year:
+# https://docs.moodle.org/311/en/Nginx
+# https://github.com/moodlebox/moodlebox/blob/master/roles/webserver/templates/etc/nginx/sites-available/default.j2
+
+# Odd that MoodleBox.net is using only the 2nd half of https://docs.moodle.org/311/en/Nginx#XSendfile_aka_X-Accel-Redirect ?
+#location ~ ^/moodle/dataroot/ {
+#    internal;
+#    alias {{ moodle_data }};
```

Comment:
As stated in Moodle documentation, ensure this path ends with a slash

Reply:
yup noted https://docs.moodle.org/311/en/Nginx